### PR TITLE
ui: Add radio option for log format to K8s log integration

### PR DESCRIPTION
**Pull request by nickbp**
Adds an option for specifying whether the cluster is running CRI/containerd or dockerd. This is required by the underlying log scraper - depending on the runtime, the paths for the data are different and they have totally different formats. Previously, the code was hardcoded to CRI/containerd even though the underlying yaml template had an option for dockerd. There was just a TODO for exposing this as an option.

The result looks like this. Could probably be cleaned up a bit but I figured that this should be okay for now (baby's first react UI):
k8slogs-radio-help

### CODE REVIEW

**Comment by nickbp** [comment]
@terrcin LMK if you have any pointers for handling the margins better in the help text. Is there a component or option for doing this? I wanted the output to look like the above screenshot, but I imagine the underlying `&nbsp;` hack isn't the correct way to accomplish this.

File: packages/app/src/client/integrations/k8sLogs/Form.tsx [hunk 1]
```diff
@@ -22,24 +22,29 @@ import * as yup from "yup";
...
+  deployNamespace: yup.string().required(),
+  format: yup.string().required()
 });
 
 const defaultValues: Values = {
   name: "",
-  deployNamespace: "opstrace"
+  deployNamespace: "opstrace",
+  format: "cri"
 };
```
Comment (MoSattler) [inline packages/app/src/client/integrations/k8sLogs/Form.tsx:48]: You can DRY this a bit up my deriving `Values` from `yup`.

```ts
const Schema = yup.object({
  name: yup.string().required(),
  deployNamespace: yup.string().required(),
  format: yup.string().required()
});

type Values = yup.TypeOf<typeof Schema>
```

**Review by MoSattler — COMMENTED** [review]
Some typing suggestions!

**Comment by nickbp** [comment]
Thanks! I've added the typing improvements and will check it once more before merging (depending on how CI goes)

**Comment by nickbp** [comment]
GCP failed on `view_ring_health` in test-remote-ui, known-flaky.

Tested the UI changes by hand, and things worked as expected - the radio option worked as expected, the UI layout looked fine, and CRI and dockerd-format yaml configs were rendered correctly

**Pull request merged**